Berlin/BerHsOr9834.xml

```diff
@@ -70,10 +70,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
                </physDesc>
                <history>
                    <origin>
-                      <origPlace> <placeName ref="LOC2422DagaE"/> </origPlace> 
+                      <origPlace> <placeName ref="LOC2422DagaEs"/> Dāgā ʾƎsṭifānos </origPlace> 
```

Comment:
`<placeName ref="LOC2422DagaEs"/> Dāgā ʾƎsṭifānos `: same remark on the double rendering as below.